ci/src/cI_web_templates.ml

```diff
@@ -89,7 +89,7 @@ let escape_ref path =
   Uri.pct_encode ~scheme:"http" (String.concat ~sep:"/" path)
 
 let unescape_ref s =
-  Uri.pct_encode ~scheme:"http" s |> (fun s -> String.cuts ~sep:"/" s)
+  Uri.pct_decode s |> (fun s -> String.cuts ~sep:"/" s)
```

Comment:
Actually I am not sure to understand. We used to decode and the split in d1fc7b7, I'm just re-doing the same thing here.